components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemTransfer.vue

```diff
@@ -1,88 +1,55 @@
 <template>
-  <div>
-    <Loader v-model="isLoading" :status="status" />
-    <div class="is-flex gallery-item-transfer">
-      <GalleryItemActionSlides ref="actionRef" :active="active">
-        <template #action>
-          <NeoTooltip
-            :active="isTransferButtonDisabled"
-            append-to-body
-            multiline
-            position="bottom"
-            :label="$t('tooltip.emptyAddress')">
-            <NeoButton
-              :label="`${$i18n.t('transaction.transfer')}`"
-              size="large"
-              fixed-width
-              :disabled="isTransferButtonDisabled"
-              no-shadow
-              @click.native="sendItem" />
-          </NeoTooltip>
-        </template>
-
-        <template #content>
-          <input
-            v-model="address"
-            type="text"
-            class="px-4"
-            :placeholder="`${$i18n.t('transaction.transferTo')}:`" />
-        </template>
-      </GalleryItemActionSlides>
-    </div>
+  <div class="is-flex is-justify-content-end gallery-item-transfer">
+    <NeoButton
+      :label="`${$i18n.t('transaction.transfer')}`"
+      size="large"
+      variant="k-pink"
+      fixed-width
+      @click.native="sendItem" />
+
+    <ItemTransferModal
+      v-model="isModalActive"
+      :nft="nft"
+      @close="isModalActive = false" />
   </div>
 </template>
 
 <script setup lang="ts">
-import { onClickOutside } from '@vueuse/core'
-import { NeoButton, NeoTooltip } from '@kodadot1/brick'
-
-import GalleryItemActionSlides from '../GalleryItemActionSlides.vue'
-import { Interaction } from '@kodadot1/minimark/v1'
+import { NeoButton } from '@kodadot1/brick'
 
-const { transaction, status, isLoading } = useTransaction()
-const { $route, $i18n } = useNuxtApp()
-const { urlPrefix } = usePrefix()
+import ItemTransferModal from '@/components/common/itemTransfer/ItemTransferModal.vue'
+import { NFT } from '@/components/rmrk/service/scheme'
 
-const props = defineProps<{
-  nftId: string
+defineProps<{
+  nft: NFT
 }>()
 
-const active = ref(false)
-const address = ref()
-
-const isTransferButtonDisabled = computed(() => {
-  return active.value && !address.value
-})
+const isModalActive = ref(false)
 
 function sendItem() {
-  if (active.value === false) {
-    active.value = true
-  } else {
-    transaction({
-      interaction: Interaction.SEND,
-      urlPrefix: urlPrefix.value,
-      address: address.value,
-      tokenId: $route.params.id,
-      nftId: props.nftId,
-      successMessage: $i18n.t('transaction.item.success') as string,
-      errorMessage: $i18n.t('transaction.item.error') as string,
-    })
-  }
+  isModalActive.value = true
 }
-
-const actionRef = ref(null)
-onClickOutside(actionRef, () => (active.value = false))
 </script>
 
 <style lang="scss" scoped>
 @import '@/styles/abstracts/variables';
 .gallery-item-transfer {
-  justify-content: flex-end;
+  display: flex;
+
+  button {
+    font-size: 1rem;
+    height: 3.375rem;
+  }
```

Comment:
I would use `size` prop

Reply:
how prop size large looks 
![CleanShot 2023-09-12 at 12 27 58@2x](https://github.com/kodadot/nft-gallery/assets/44554284/756dfa4e-ddf0-458d-a9b8-70f3e13d9796)

the height is the same as the large button 54px

how it should look 
<img width="213" alt="CleanShot 2023-09-12 at 12 33 43@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/c4a00b91-d3bf-4e2a-968e-be9fe156ce85">